azkaban-common/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -40,28 +41,41 @@
 @Singleton
 public class MetricsManager {
 
-  private static final Logger logger = Logger.getLogger(MetricsManager.class);
+  private static final Logger log = LoggerFactory.getLogger(MetricsManager.class);
   private final MetricRegistry registry;
-  private ConsoleReporter consoleReporter = null;
 
-  /**
-   * Constructor is eaagerly called when this class is loaded.
-   */
   @Inject
-  public MetricsManager(MetricRegistry registry) {
+  public MetricsManager(final MetricRegistry registry) {
     this.registry = registry;
+    registerJvmMetrics();
+  }
+
+  private void registerJvmMetrics() {
     this.registry.register("MEMORY_Gauge", new MemoryUsageGaugeSet());
     this.registry.register("GC_Gauge", new GarbageCollectorMetricSet());
     this.registry.register("Thread_State_Gauge", new ThreadStatesGaugeSet());
   }
 
   /**
-   * Return the Metrics registry.
+   * A {@link Meter} measures the rate of events over time (e.g., “requests per second”).
+   * Here we track 1-minute moving averages.
+   */
+  public Meter addMeter(final String name) {
```

Comment:
+1 on moving static method to here. We could modify the comment following the actual method below `getFifteenMinuteRate `